src/main/java/org/gridsuite/study/server/service/ReportService.java

```diff
@@ -90,7 +91,7 @@ private ReportNode reportServerCall(UUID id, String serverUri, UriComponentsBuil
         var path = uriBuilder.buildAndExpand(id).toUriString();
         HttpHeaders headers = new HttpHeaders();
         headers.setContentType(MediaType.APPLICATION_JSON);
-        List<ReportNode> reporters = restTemplate.exchange(serverUri + path, HttpMethod.GET, new HttpEntity<>(headers), new ParameterizedTypeReference<List<ReportNode>>() {
+        List<ReportNodeImpl> reporters = restTemplate.exchange(serverUri + path, HttpMethod.GET, new HttpEntity<>(headers), new ParameterizedTypeReference<List<ReportNodeImpl>>() {
```

Comment:
strange, I as persuaded that Jackson handled correctly interfaces when deserializing...

Well just need to update the reference then:
```suggestion
        List<ReportNode> reporters = restTemplate.exchange(serverUri + path, HttpMethod.GET, new HttpEntity<>(headers), new ParameterizedTypeReference<List<ReportNodeImpl>>() {
```

Reply:
Same thing has been done in powsybl-core, in this PR : https://github.com/powsybl/powsybl-core/pull/3011
when they upgrade to ReportNode V2.1 ...